raster/src/main/scala/geotrellis/raster/io/geotiff/StorageMethod.scala

```diff
@@ -51,3 +53,47 @@ object Striped extends StripedStorageMethod {
 
   implicit def objectToStorageMethod(s: StripedStorageMethod): Striped = Striped()
 }
+
+object StorageMethod {
+  implicit val storageMethodDecoder: Decoder[StorageMethod] =
+    new Decoder[StorageMethod] {
+      final def apply(c: HCursor): Decoder.Result[StorageMethod] = {
+        for {
+          storageType <- c.downField("storageType").as[String]
+        } yield {
+          storageType match {
+            case "striped" => {
+              val rowsPerStrip =
+                c.downField("rowsPerStrip").as[Option[Int]] match {
+                  case Left(_)  => None
+                  case Right(v) => v
+                }
+              new Striped(rowsPerStrip)
+            }
+            case _ => {
+              val cols = c.downField("cols").as[Int] match {
+                case Left(_)  => 256
+                case Right(v) => v
+              }
+              val rows = c.downField("rows").as[Int] match {
+                case Left(_)  => 256
+                case Right(v) => v
+              }
+              Tiled(cols, rows)
+            }
+          }
+        }
+      }
+    }
+
+  implicit val storageMethodEncoder: Encoder[StorageMethod] =
+    new Encoder[StorageMethod] {
+      final def apply(a: StorageMethod): Json = a match {
+        case _: Striped => Json.obj(("storageType", Json.fromString("striped")))
```

Comment:
`value.asJson` is more readable rather than these `Json.fromType` // just a minor readability notion. 

requires an `io.crice.syntax._` though.